ts/features/bonus/bpd/saga/index.ts

```diff
@@ -37,15 +43,14 @@ export function* watchBonusBpdSaga(bpdBearerToken: string): SagaIterator {
   // load citizen details
   yield takeLatest(
     bpdLoadActivationStatus.request,
-    getCitizen,
-    bpdBackendClient.find
+    bpdTechnicalIban ? getCitizenV2 : getCitizen,
+    bpdBackendClient.findV2
```

Comment:
```suggestion
    bpdTechnicalIban ? bpdBackendClient.findV2 : bpdBackendClient.find
```
Even if the signature is the same, I think it is better to keep the distinction between the two different versions for the client as well, to avoid confusion in the cleaning phase after removing the feature flag, what do you think?

Reply:
You've surely more experience regarding this situation.
I followed the suggestion of Undermaken in the previous comment because I thought that in this way we have only one entry point as well as having less code.

But if you think that is less confusing I revert the commit :)

Reply:
I think in this phase is better to keep the original endpoint until the feature flag is remove, in order to keep the actual functionality until we are sure that the new endpoints work like expected in production. For example if we are going to release a new app version with the feature flag disabled, the `getCitizen` will uses the new endpoint with unexpected behaviour (some endpoints /v2 are still not implemented in productions).

Reply:
Sorry, I misunderstood the comment! 

You're right!

Reply:
Modified in [a6d8119](https://github.com/pagopa/io-app/pull/2969/commits/a6d81191d57f6222c2b125346eff0afca5ec8829)